internal/impl/sql/integration_test.go

```diff
@@ -30,19 +31,19 @@ func testProcessors(name string, fn func(t *testing.T, insertProc, selectProc se
 	return func(t *testing.T, driver, dsn, table string) {
 		t.Run(name, func(t *testing.T) {
 			insertConf := fmt.Sprintf(`
-driver: %v
-dsn: %v
-table: %v
-columns: [ foo, bar, baz ]
+driver: %s
+dsn: %s
+table: %s
+columns: [ "\"foo\"", "\"bar\"", "\"baz\"" ]
```

Comment:
Unlike other drivers, Oracle returns the columns as upper case if they're not explicitly created as lower case in the `create table` statement. If I change all these columns to upper case, then it breaks the other providers...